### ISSUE TITLE
Pascal scripting interface does not work

### ISSUE THREAD

**Issue by ToeKneeFan** [neurolabusc/surf-ice#31]
I am not sure if this is a system/environment-dependent issue or an actual bug, but I thought I would report just in case. Pascal scripts that once worked in old versions of Surfice appear to no longer run in newer versions of Surfice. If I try to run even simple scripts like the following, nothing occurs:

```
BEGIN
resetdefaults;
meshload('BrainMesh_ICBM152_smoothed.lh.mz3')
END.
```

I tested on version 6-October-2021 on NITRC (as well as the Github copy). If I reinstall 2-November-2020, the scripts work fine (although the software itself suffers from issues that have since been addressed through 6-October-2021, so staying on the older version is not ideal). 

Perhaps it is due to related to integration of PythonBridge or Python-for-Lazarus?

I am on macOS Big Sur 11.6.4.

**Comment by neurolabusc** [comment]
Is this on a M1 (Apple Silicon) CPU? Older versions of Surfice run on M1 via Rosetta emulation, the latest versions run natively. PascalScript does not support the ARM architecture. For this reason, I am retiring the PascalScript support.

**Comment by ToeKneeFan** [comment]
This issue appears to also manifest on the Windows version as well. If I had to guess, with the introduction of PythonBridge, the logic that chooses or implements Pascal/Python scripting was broken. 

If you prefer, I can just close this issue, since the Pascal scripting interface will be retired anyway. For now, I'll leave open in case there are plans to maintain Pascal support. For what it is worth, I ended up just migrating to Python.
